rpc_jobs/rpc_gating_lint.yml

```diff
@@ -1,6 +1,7 @@
 - job:
     name: 'RPC-Gating-Lint'
     project-type: workflow
+    concurrent: true
```

Comment:
concurrent defaults to `false`, and there is no reason why this job cannot run concurrently.